### PR TITLE
add screening

### PR DESCRIPTION
**Pull request by ridzikowski**
move screening from travel branch

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -19,4 +19,4 @@ Setfield = "efcf1570-3423-57d1-acb7-fd33fddbac46"
 StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 
 [compat]
-MocosSim = "~0.3"
+MocosSim = "0.3.6"
```
Comment (tozanski) [inline Project.toml:22]: this change is not needed. Before the change it was already compatible with all the 0.3.x versions so only the update was needed

File: example/example.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
     "probability" : 0.5,
     "backward_detection_delay" : 1.75,
     "forward_detection_delay" : 1.75,
-    "testing_time" : 0.25
+    "testing_time" : 3.0
```
Comment (tozanski) [inline example/example.json:9]: stray change

File: Project.toml (hunk 1 above)
Comment (ridzikowski) [reply Project.toml:22]: yes, the update fixed the issue with the version

File: src/MocosSimLauncher.jl [hunk 3]
```diff
@@ -93,14 +93,16 @@ function launch(args::AbstractVector{T} where T<:AbstractString)
...
   end
-
   @threads for trajectory_id in 1:num_trajectories
     state = states[threadid()]
     MocosSim.reset!(state, trajectory_id)
     
     for fun in import_funcs
       fun(state, params)
     end
+    if params.screening_params != nothing
```
Comment (tozanski) [inline src/MocosSimLauncher.jl:103]: compare to nothing with `!==` and `===`or with `isnothing`
```suggestion
    if params.screening_params !== nothing
```

**Review by tozanski — APPROVED** [review]
LGTM (please add my sugestion before merging)

**Review by tozanski — APPROVED** [review]

**Pull request merged**